src/main/java/crazypants/enderio/BlockEio.java

```diff
@@ -95,6 +95,10 @@ public boolean onBlockActivated(World world, int x, int y, int z, EntityPlayer e
       }
     }
     
+    if (entityPlayer.isSneaking()) {
```

Comment:
I think this check should be integrated into the next if clause (i.e. `if (!entityPlayer.isSneaking() && te instanceof ITankAccess)`) so that it does not interfere with later code that might want to function if the player is sneaking.